dts/bindings/iio/adc/nuvoton,npcx-adc.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-ADC node
+
+compatible: "nuvoton,npcx-adc"
+
+include: [adc-controller.yaml, base.yaml]
```

Comment:
Including `base.yaml` is redundant here, it's already included by `adc-controller.yaml`

Reply:
Done.